src/main/java/game/jdbc/picturepoker/PicturePokerGame.java

```diff
@@ -1,7 +1,7 @@
 package game.jdbc.picturepoker;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
-import jdk.internal.misc.InnocuousThread;
+// import jdk.internal.misc.InnocuousThread; // Why is this here?
```

Comment:
This is from the sample code provided in class - I also am not too certain what it does.